packages/vanchor-client/src/graphql/tvl.graphql

```diff
@@ -175,3 +175,22 @@ query GetVAnchorsTotalValueLockedByDateRange(
     endInterval
   }
 }
+
+query GetVAnchorLatestTVLInTimeRange(
```

Comment:
Rename the query to `GetFirstVAnchorLatestTVLInTimeRange` to indicate the query only returns 1 result.